django/core/signing.py

```diff
@@ -159,13 +159,16 @@ def loads(
     serializer=JSONSerializer,
     max_age=None,
     fallback_keys=None,
+    expiration_key=None,
```

Comment:
`loads()` signature should be updated in [docs](https://docs.djangoproject.com/en/dev/topics/signing/#protecting-complex-data-structures) in a `versionchanged` annotation.

Reply:
I'd add tests for `loads()` calls with custom `expiration_key`.